data/json/mapgen/refugee_center.json

```diff
@@ -5,6 +5,7 @@
     "om_terrain": [ [ "evac_center_1", "evac_center_2", "evac_center_3", "evac_center_4", "evac_center_5" ] ],
     "weight": 100,
     "object": {
+      "faction_owner": [ { "id": "free_merchants", "x": [ 0, 119 ], "y": [ 0, 23 ] } ],
```

Comment:
is 119 a typo?

Reply:
I think it should be 19.

It breaks tests now:

![image](https://user-images.githubusercontent.com/16213433/57577433-552a1280-747f-11e9-8dc1-6957ac1f093d.png)

Reply:
I changed it, when I saw the merge conflicts, and the evac_centre.json map had changed to a new file.
I saw this new file had a much larger width, I assumed, it would be a matter of extending the zone to match the new wider style of maps?
Before it was 0, 23....

I guess not however. Not entirely sure what I can do about that, as I barely understood the mapgen part of this code at the best of times, and I'm not sure what the deal is with these wider maps?

Reply:
sorted